openverse_catalog/dags/storage/columns.py

```diff
@@ -273,3 +576,111 @@ def prepare_string(self, value):
                 values.append(self.BASE_COLUMN.prepare_string(val))
         arr_str = json.dumps(values, ensure_ascii=False)
         return "{" + arr_str[1:-1] + "}" if arr_str else None
+
+
+FOREIGN_ID_COLUMN = StringColumn(
```

Comment:
The `_COLUMN` suffix on these feels a bit redundant, particularly seeing how this is used in other files. Do you think it would be okay to drop that for these values?

Reply:
The names without `_COLUMN` prefix were used in `column_names.py` files, so I used the prefix here. Fixed it in the new commit, by removing `column_names.py`, and using column definitions from this file everywhere (some files became more verbose, but they can be improved later, with automatic query creation)